pkg/service/sync.go

```diff
@@ -88,6 +89,10 @@ func (s *SyncService) SyncRewardHistory(ctx context.Context, exchange types.Exch
 	}
 
 	log.Infof("syncing %s reward records...", exchange.Name())
+	if util.IsPaperTrade() {
+		log.Info("Reward is not supported in paper trading")
```

Comment:
such check is done by checking the interface, you can check pkg/types/exchange.go

Reply:
there is no check there
In fact, when I don't have that code, sync progress is corrupted with error below

```
[0001]  INFO syncing binance reward records...
Error: request failed with status code: 404, body: ""
[0002] FATAL cannot execute command error=request failed with status code: 404, body: ""
```

Reply:
inside the QueryRewardHistory method, return empty slice for paper trading? and actually you can turn off the reward history sync ;-)

Reply:
inside the QueryRewardHistory method --> there is no method with name QueryRewardHistory

and actually you can turn off the reward history sync  --> you mean there is env to on/off this? ah, I found it, ```rewardHistory: true```

At last, I think this code is ok at this time, just because it has the same pattern with deposit / withdrawn methods